components/style/gecko/selector_impl.rs

```diff
@@ -70,7 +70,7 @@ impl PseudoElement {
             }}
         }
 
-        include!("generated/gecko_pseudo_element_helper.rs");
+        include!("../generated/gecko_pseudo_element_helper.rs");
```

Comment:
we can put this file in the `gecko` directory now, something like `gecko/generated/pseudo_element_helper.rs`.